parm/config/gfs/config.resources

```diff
@@ -113,7 +113,11 @@ case ${step} in
     export nth_waveinit=1
     export npe_node_waveinit=$(( npe_node_max / nth_waveinit ))
     export NTASKS=${npe_waveinit}
-    export memory_waveinit="2GB"
+    if [[ "${machine}" == "GAEA" ]]; then
+      export memory_waveinit=""
```

Comment:
Instead of doing this for every task, can you try this at the end of `config.resources` while we get feedback from the Gaea sysadmins:
```bash
if [[ "${machine}" == "GAEA" ]]; then
  for mem_var in $(env | grep '^memory_' | cut -d= -f1); do
    unset "${mem_var}"
  done
fi
```
It should unset all `memory_` variables for Gaea without having to do so for each task.

Reply:
Morning @aerorahul. I'm responding to the ticket right now and will test this out today. Thanks.